README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+# About Warren Casey Epstein
+## Undergraduate Student at the University of Chicago
+
+
+## Rising Third-Year in the College
```

Comment:
In the future, you should try to leave a blank line after headers. It's not an issue here, but once you begin working with code chunks in RMarkdown files later, this practice has the potential to save you some formatting troubles.